python/instrumentation/openinference-instrumentation-langchain/src/openinference/instrumentation/langchain/__init__.py

```diff
@@ -68,3 +77,30 @@ def __call__(
                 break
         else:
             instance.add_handler(self._tracer, True)
+
+
+def get_current_span() -> Optional[Span]:
+    import langchain_core
+
+    run_id: Optional[UUID] = None
+    config = langchain_core.runnables.config.var_child_runnable_config.get()
+    if not isinstance(config, dict):
+        return None
+    for v in config.values():
+        if not isinstance(v, langchain_core.callbacks.BaseCallbackManager):
+            continue
+        if run_id := v.parent_run_id:
+            break
+    if not run_id:
+        return None
+    return LangChainInstrumentor().get_span(run_id)
+
+
+def __getattr__(name: str) -> Any:
+    if name == get_current_span.__name__:
+        if tuple(map(int, version("langchain-core").split(".")[:3])) < (0, 1, 47):
+            raise ImportError("get_current_span requires langchain-core>=0.1.47")
+        return get_current_span
+    if name == LangChainInstrumentor.__name__:
+        return LangChainInstrumentor
+    raise AttributeError(name)
```

Comment:
Can we move this near the top of the module?

Reply:
I think this is somewhat ancillary, so it shouldn't be first thing a reader sees when the file is opened.